s2a_options.go

```diff
@@ -125,6 +145,12 @@ type ClientOptions struct {
 
 	// Optional fallback after dialing with S2A fails.
 	FallbackOpts *FallbackOptions
+
+	// Generates S2AStream for talking to the S2A server.
+	GetS2AStream *func() (*S2AStream, error)
+
+	// Serialized user specified policy for server authorization.
+	serverAuthorizationPolicy []byte
```

Comment:
this probably should be exported as well

Reply:
Per offline discussion, all newly added fields will be unexported.